src/Microsoft.DotNet.ProjectModel/Files/PackIncludeEntry.cs

```diff
@@ -26,18 +32,16 @@ public PackIncludeEntry(string target, string[] sourceGlobs, int line, int colum
             Column = column;
         }
 
-        private static string[] ExtractValues(JsonValue json)
+        private static string[] ExtractValues(JToken json)
```

Comment:
I tried to refactor as little as possible. This was a big enough change without that already. A whole set of improvements could and should be made here, specially now that we have test coverage. But I don't want to do that as part of this change, as it would take more time and asp.net is blocked waiting for this change.